optimade/server/tests/test_structures.json

```diff
@@ -165,7 +172,14 @@
       "Ir"
     ],
     "structure_features": [],
-    "task_id": "mpf_2"
+    "task_id": "mpf_2",
+    "relationships": {
+        "references": {
+            "data": [
+                {"type": "references", "id": "Dijkstra1968"}
```

Comment:
These relationships should be added to the reference datum as well.

Reply:
True, I guess the reference endpoint should support the opposite. This is going to lead to every endpoint having to query every other endpoint per query though...

Reply:
Indeed, we should bring this up at the meeting today: Either we should clarify one-way relationships or both-way relationships.
In my interpretation from JSON API though, it seems it goes both ways. This makes sense from the point-of-view that one has found a specific reference as well and now want to see all structures mentioned/published in this reference...